components/modal/header.js

```diff
@@ -0,0 +1,29 @@
+import IconButton from '../icon-button';
+import './style.scss';
+import { __ } from '@wordpress/i18n';
+
+const ModalHeader = ( { icon, title, onClose, closeLabel } ) => {
+	const label = closeLabel ? closeLabel : __( 'Close window' );
```

Comment:
A better default text would be `Close dialog` (when a dialog opens, screen readers announce `dialog` or `web dialog`), or simply `Close`.